imports/ui/pages/post/PostList.jsx

```diff
@@ -69,10 +80,21 @@ export default class PostList extends React.Component {
                         _.map(posts, (post) => {
                             return (
                                 <li key={post._id}>
+
                                     <div>{post.title}</div>
                                     <div>{post.description}</div>
                                     <div>{moment(post.createdAt).format('DD MMM YYYY')}</div>
                                     <div>{post.userId}</div>
+                                    <div>{post.postLikesCount}</div>
+                                    {
+                                        post.postLikes
+                                            ?
+                                            <button type="button" className="btn btn-default"
+                                                    onClick={this.addAndRemoveLike.bind(this, post._id)}>Unlike</button>
```

Comment:
If the only difference between these is the name of action, you could make a variable, const likeAction = post.postLike ? Unliked : Like